data/publiccloud/terraform/azure.tf

```diff
@@ -0,0 +1,146 @@
+provider "azurerm" {}
+
+variable "count" {
+    default = "1"
+}
+
+variable "name" {
+    default = "openqa-vm"
+}
```

Comment:
Wow, do you think they could have made this more verbose? I prefer more characters in my variable definitions. LOL